webapp/components/Avatar/Avatar.vue

```diff
@@ -1,5 +1,10 @@
 <template>
-  <ds-avatar :image="avatarUrl" :name="userName" class="avatar" :size="size" />
+  <ds-avatar
+    :image="user && user.avatar | proxyApiUrl"
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-06-13T23:55:10Z" title="Friday, June 14th 2019, 1:55:10 am +02:00">Jun 14, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,5 +1,10 @@
 <template>
-  <ds-avatar :image="avatarUrl" :name="userName" class="avatar" :size="size" />
+  <ds-avatar
+    :image="user && user.avatar | proxyApiUrl"
```


I needed to check what the `|` means in this case @roschaefer 
it's a bitwise OR, correct?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Bitwise_Operators#.7c_%28Bitwise_OR%29
can you explain why you used it in this case?

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-06-14T07:48:09Z" title="Friday, June 14th 2019, 9:48:09 am +02:00">Jun 14, 2019</time>_

---


No, this is not a bitwise OR. It's the use of a vue filter in a template. See https://vuejs.org/v2/guide/filters.html